statsmodels/iolib/foreign.py

```diff
@@ -263,6 +265,11 @@ class StataReader(object):
             (-1.798e+308, +8.988e+307) }
 
     def __init__(self, fname, missing_values=False, encoding=None):
+        warnings.warn(
+            "StataReader is deprecated as of 0.10.0 and will be removed in a "
+            "future version.  Use pandas.read_stata instead.",
```

Comment:
Ideally something mentions both read_stata and StataReader, e.g., use pandas.read_state or pandas.io.stata.StataReader